HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
@adcxyz I'd be for having this kind of tricks in a special "Tips and Tricks" help file with all kind of syntactic sugar and what not instead of adding too much info in an unrelated HelpFile?